### DISCUSSION TITLE
Add Single VFO Display

### DISCUSSION THREAD

**Discussion by baumertjohn** [egzumer/uv-k5-firmware-custom#187] [Ideas]
Would it be possible to add "single" display when only monitoring one station?  This was recently added to the OneOfEleven build - ( https://github.com/OneOfEleven/uv-k5-firmware-custom/commit/1bff1b5cb1d0978e75810f644b6463b7fc86a190 )

**Comment by ErikS-web** [comment]
Your idea is already posted. See #124

**Comment by baumertjohn** [comment]
I did see that feature.  I was curious if it would be possible to implement the "single channel display" specifically when having single channel RX selected.
